price-feeder/oracle/oracle.go

```diff
@@ -33,15 +42,271 @@ func (o *Oracle) Stop() {
 	<-o.closer.Done()
 }
 
-func (o *Oracle) Start(ctx context.Context) {
+var denoms = []string{"ATOMUSDT"}
+
+// Should return a prices map that we can set
+// The oracle to. Needs to touch multiple providers,
+// And then average out.
+func GetPrices() (map[string]sdk.Dec, error) {
+
+	binanceProvider := provider.NewBinanceProvider()
+	krakenProvider := provider.NewKrakenProvider()
```

Comment:
A few thoughts on this one:

1. Godocs should start with the name, e.g. `// GetPrices ...`
2. We shouldn't assume anything about providers. See my previous on the constructor. The operator of the price-feeder provides what providers they want to use.

Based on (2) I imagine this method would completely change